tests/suites/test_suite_psa_crypto.data

```diff
@@ -6320,6 +6320,34 @@ PSA key derivation: HKDF-Expand SHA-1, request too much capacity
 depends_on:PSA_WANT_ALG_HKDF_EXPAND:PSA_WANT_ALG_SHA_1
 derive_set_capacity:PSA_ALG_HKDF_EXPAND(PSA_ALG_SHA_1):255 * PSA_HASH_LENGTH(PSA_ALG_SHA_1) + 1:PSA_ERROR_INVALID_ARGUMENT
 
+PSA key derivation: TLS 1.2 PSK-to-MS, request too much capacity
+depends_on:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PSK_TO_MS
+derive_set_capacity:PSA_ALG_TLS12_PSK_TO_MS(PSA_ALG_SHA_256):48U + 1U:PSA_ERROR_INVALID_ARGUMENT
```

Comment:
For algorithms that are parametrized by a hash algorithm, please make sure that there are test cases with at least two hashes, to validate that the hash is taken into account. Missing for `derive_set_capacity` for the algorithms other than HKDF/HKDF-EXPAND/HKDF-EXTRACT. Missing for `derive_full` for all relevant algorithms.